target/linux/layerscape/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,12 @@ case "$(board_name)" in
 		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth6"
 		ucidef_set_interface_wan "eth4"
 		;;
+	fsl,ls1028a-rdb | \
+	fsl,ls1028a-rdb-sdboot)
+		ip link set eno2 up
```

Comment:
This script isn't launched at every boot.

It should be moved to preinit scripts. Examples:

https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/mt7621/base-files/lib/preinit/07_mt7621_bringup_dsa_master

https://github.com/openwrt/openwrt/blob/master/target/linux/mediatek/base-files/lib/preinit/05_set_preinit_iface

Reply:
Done.